swagger.yml

```diff
@@ -935,6 +956,7 @@ components:
             - NEW_LICENSE
             - NEW_SERVER
             - NO_DATA
+          description: '"NEW_DATABASE", "NEW_OPTION", "NEW_LICENSE", "NEW_SERVER", "NO_DATA"'
```

Comment:
Is the description redundant to the enum property? This don't seem a description to me

Reply:
I have removed the description. The enum was not visible from the StopLight view